scripts/serve.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
Add license header, as well as the source SO link.

Reply:
Done!